cylc/flow/tests/parsec/test_fileparse.py

```diff
@@ -224,7 +237,9 @@ def test_multiline(self):
                 with self.assertRaises(exc) as cm:
                     multiline(flines, value, index, maxline)
                 if isinstance(cm.exception, FileParseError):
-                    self.assertEqual(expected, str(cm.exception))
+                    exc = cm.exception
+                    for key, value in expected.items():
```

Comment:
could we name this `value` as `value2` or something else? There is an outer loop here declaring it already `for flines, value, ` ...